packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -879,7 +880,9 @@ const StepNumberCompleted = styled( StepNumber )`
 	// prettier-ignore
 	@media all and (-ms-high-contrast:none), (-ms-high-contrast:active) {
 		backface-visibility: visible;
-		z-index: ${ ( props ) => ( props.isComplete ? '1' : '0' ) };
+		z-index: ${ (
+		props
```

Comment:
I'm not sure this is necessary either, and I think it might actually cause issues. What was the reason for this change?